sktime/transformations/panel/summarize/_extract.py

```diff
@@ -17,18 +22,53 @@
 class PlateauFinder(BaseTransformer):
     """Plateau finder transformer.
 
-    Transformer that finds segments of the same given value, plateau in
-    the time series, and
-    returns the starting indices and lengths.
+    Transformer that finds segments of the same given value, aka plateau(s) in
+    the time series, and returns the starting indices and lengths. It expects a
+    dataframe with one column of nested Series and it extracts the plateau for
+    each series. The output is a dataframe with two columns of nested lists,
+    where the first column has the lists with the starting indices of the
+    plateau(s), and the second column has the lists of the length of the
+    plateau(s). If no flate segements are found, the output is a dataframe of
+    nested empty lists.
+
+    In this context the `value` parameter tells the transformer to look for flat
+    segments with a specific value, the `min_lenght` tells to filter
+    plateau(s) with a minimum length, and the `max_length` parameter, which is
+    optional, tells the transformer to filter for segments with a specific
+    maximum length.
 
     Parameters
     ----------
     value : {int, float, np.nan, np.inf}
         Value for which to find segments
     min_length : int
-        Minimum lengths of segments with same value to include.
+        Minimum length of segments with same value to include, where the
+        length is number of consecutive samples within a segment.
         If min_length is set to 1, the transformer can be used as a value
         finder.
+    max_length : int
+        Maximum length of segments with same value to include, where the
+        length is number of consecutive samples within a segment. If max_length
+        is set to None, a maximum segment length filter is not applied to
+        segments
+
+    Examples
+    --------
+    >>> # check out https://github.com/alan-turing-institute/sktime/blob/main/examples/plateau_finder.ipynb # noqa: E501
+    >>> import pandas as pd
+    >>> from sktime.transformations.panel.summarize import PlateauFinder
+    >>> # generate some data
+    >>> X = pd.DataFrame(
+            pd.Series([
+                pd.Series([-1, -1, 3, 3, -1, 2, 2, 3]),
+                pd.Series([0, -1, -1, -1, -1, -1, 2, -1]),
+                pd.Series([2, -1, -1, -1, 2, -1, 3, 1]),
+                pd.Series([1, -1, -1, 3, -1, -1, 2, 0]),
+            ])
+        )
```

Comment:
Tests are failing because of this example. If you add `...` in all indented rows it should work. 
e.g. [example syntax for doc examples](https://github.com/alan-turing-institute/sktime/blob/3194e3ee961cdb47bac3b487a779582b134eab68/sktime/transformations/bootstrap/_mbb.py#L154) 
To test the examples locally you can run the following command in your terminal: 
`python -m doctest -o=ELLIPSIS -v sktime/transformations/panel/summarize/_extract.py`

Reply:
> To test the examples locally

Oh, I did not know that, @ltsaprounis!
This would be useful in our "adding a new estimator" or "testing" dev documentation.

Reply:
thx! fixed in [a4d2eae](https://github.com/alan-turing-institute/sktime/pull/2278/commits/a4d2eae6a1db0f959c6d821e2c9864d8d71da507)

Reply:
Yep! that's done

Reply:
Yep! that's done